dependency_support/com_google_skywater_pdk/declare_cell_library.bzl

```diff
@@ -42,25 +35,22 @@ def declare_cell_library(library_name):
     library = CELL_LIBRARIES[library_name]
     corners = library.get("corners", {})
     for corner in corners:
-        corner_types = library["corners"][corner]
-        for corner_type in corner_types:
-            out_path = "timing/%s__%s%s.lib" % (library_name, corner, _FILE_SUFFIX_BY_CORNER_TYPE[corner_type])
-            native.genrule(
-                name = "timing_%s_%s" % (corner, corner_type),
-                visibility = ["//visibility:public"],
-                outs = [out_path],
-                srcs = native.glob(["**/*"]),
-                message = "Generating %s timing files, %s corner %s" % (library_name, corner_type, corner),
-                cmd = "$(location @com_google_skywater_pdk//:liberty) %s \"external/com_google_skywater_pdk_%s\" %s > /dev/null && mv \"external/com_google_skywater_pdk_%s/%s\" $@" % (_GENERATOR_ARGUMENT_BY_CORNER_TYPE[corner_type], library_name, corner, library_name, out_path),
-                tools = ["@com_google_skywater_pdk//:liberty"],
-            )
-        native.filegroup(
-            name = "timing_%s_all" % corner,
-            visibility = ["//visibility:public"],
-            srcs = [":timing_%s_%s" % (corner, corner_type) for corner_type in corner_types]
+        ccsnoise = "ccsnoise" in corners[corner]
+        leakage = "leakage" in corners[corner]
+        skywater_corner(
+            name = "{}".format(corner),
+            visibility = ["//visibility:private"],
+            srcs = native.glob([
+                "cells/**/*.lib.json",
+                "timing/*.lib.json",
+            ]),
+            standard_cell_name = library_name,
+            ccsnoise = ccsnoise,
+            leakage = leakage,
```

Comment:
Optional/nit: For `ccsnoise` and `leakage` it's not obvious from the names that these are booleans that decide whether they should be generated or not. Maybe `with_ccsnoise` or `generate_ccsnoise` would be clearer names?